Resources/Prototypes/Entities/Clothing/Head/misc.yml

```diff
@@ -171,3 +171,15 @@
   - type: Clothing
     sprite: Clothing/Head/Misc/squiddy.rsi
   - type: IngestionBlocker
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatDonJuan
+  name: Don Juan
+  description: 'Oh look, it's bojack!'
```

Comment:
You still need to use ' instead of " for the linter to work, also this description still isnt valid, refer to my previous comment about it

Reply:
I didn't get the 'instead of' part, can you give me an example?

Reply:
you should use `"` instead of `'` like it is currently

Reply:
done.